src/rulesets/oas/functions/__tests__/oasPathParam.test.ts

```diff
@@ -284,15 +288,15 @@ describe('oasPathParam', () => {
       {
         code: 'path-params',
         message: `The paths \`/foo/{boo}\` and \`/foo/{bar}\` are equivalent.`,
-        path: ['paths'],
+        path: ['paths', '/foo/{bar}'],
         range: {
           end: {
```

Comment:
@P0lip How come the end block of `['paths']` is the same for `['paths', '/foo/{bar}']`?
As we're getting one "step" within, I would have expected it to change.

Reply:
I guess I'd need to pass a string to `run` rather than an actual object literal.
Otherwise ranges are, well, not really precise, because we stringify the object again.
I'll do it.